src/hooks/useAccommodationsAndOffers.tsx/api.ts

```diff
@@ -1,6 +1,6 @@
 import { SearchResults, Offer, Accommodation } from '@windingtree/glider-types/types/win';
 import axios from 'axios';
-import { getPassengersBody } from './helpers';
+import { getPassengersBody, InvalidLocationError } from './helpers';
```

Comment:
this is state variable, right?

if it is state variable, should it not be returned through the hook, instead of storing it in a helper?

Reply:
It is not a state variable. I only created a custom error to help us identify location errors